Orange/widgets/visualize/owscatterplot.py

```diff
@@ -231,7 +203,6 @@ def _add_controls(self):
         self._add_controls_axis()
         self._add_controls_sampling()
         super()._add_controls()
-        self.gui.add_widget(self.gui.JitterNumericValues, self._effects_box)
```

Comment:
Is this property (JitterNumericValues) still needed?

Reply:
I'd dissolve OWPlotGui or at least clean it. I wrote an issue (#3446).